lib/smooth/application/io/i2c/SHT30.cpp

```diff
@@ -125,15 +117,29 @@ namespace smooth::application::sensor
     }
 
     // Set repeatability mode
-    void SHT30::set_repeatability_mode(Repeatabilty mode)
+    void SHT30::set_repeatability_mode(RepeatabilityMode mode)
     {
-        repeatability_command = mode;
+        // We will only use clock stretching on Medium and Low Repeatability Measurements
+        // since the ESP32 maximum i2c-master-timeout is 13ms.  If user selects
+        // EnableMeasureHighRepeatabilityWithClockStretching it will be implemented using
+        // MeasureHighRepeatabilityWithClockStretchingDisabled
+        if ((mode == RepeatabilityMode::EnableMeasureMediumRepeatabilityWithClockStretching)
+            | (mode == RepeatabilityMode::EnableMeasureLowRepeatabilityWithClockStretching))
```

Comment:
Although this evaluates to the same, please use || instead, it prevents the implicit cast from int to bool.

```suggestion
        if (mode == RepeatabilityMode::EnableMeasureMediumRepeatabilityWithClockStretching
            || mode == RepeatabilityMode::EnableMeasureLowRepeatabilityWithClockStretching)
```